gqueries/output_elements/output_series/table_141_chp_properties/industry_chp_combined_cycle_gas_power_fuelmix_heat_production_in_chp_properties.gql

```diff
@@ -1,4 +1,15 @@
 # heat production of industry_chp_combined_cycle_gas_power_fuelmix
+# accounts for CHP heat component (industry_heat_xxx) when external coupling is activated
 
 - unit = PJ
-- query = future:UNIT(V(industry_chp_combined_cycle_gas_power_fuelmix, "output_of_useable_heat + output_of_steam_hot_water"),billions)
+- query =
+    future:UNIT(
```

Comment:
I'm not a big fan of the function `UNIT`, can't we simply use the `DIVIDE` and `BILLIONS` functions? These seem to be used more generally through the model.

Reply:
I've now applied `DIVIDE` and `BILLIONS` to the CHP queries related to this project.